mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -251,30 +337,49 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 	METHOD method_33143 sendSleepingStatus ()V
 		COMMENT Sends sleeping status action bar messages to players in this world.
 	METHOD method_33144 isSleepingEnabled ()Z
+		COMMENT {@return whether sleeping can cause the night to skip}
 	METHOD method_34679 (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2338;
 		ARG 0 pos
 	METHOD method_34680 (Lnet/minecraft/class_2338;)Z
 		ARG 1 pos
 	METHOD method_37115 shouldTickEntity (Lnet/minecraft/class_1923;)Z
+		COMMENT {@return whether to tick entities at {@code pos}}
 		ARG 1 pos
 	METHOD method_37116 isChunkLoaded (J)Z
+		COMMENT {@return {@code true} if the chunk {@code chunkPos} is loaded}
 		ARG 1 chunkPos
 	METHOD method_37117 isTickingFutureReady (J)Z
 		ARG 1 chunkPos
 	METHOD method_37118 shouldTickEntity (Lnet/minecraft/class_2338;)Z
+		COMMENT {@return whether to tick entities at {@code pos}}
 		ARG 1 pos
 	METHOD method_39223 disableTickSchedulers (Lnet/minecraft/class_2818;)V
 		ARG 1 chunk
 	METHOD method_39501 tickWeather ()V
 	METHOD method_39778 cacheStructures (Lnet/minecraft/class_2791;)V
 		ARG 1 chunk
 	METHOD method_8448 updateSleepingPlayers ()V
+	METHOD method_8469 (I)Lnet/minecraft/class_1297;
+		COMMENT {@return the entity using the entity ID, or {@code null} if none was found}
```

Comment:
Consider moving the docs to the class that defines this method than here, due to the restrictions of mapping storage right now

Reply:
Whoops